src/mediaelement.js

```diff
@@ -337,7 +346,7 @@ export default class MediaElement extends WebAudio {
      * @return {number} value A floating point value between 0 and 1.
      */
     getVolume() {
-        return this.volume || this.media.volume;
+        return this.volume;
```

Comment:
this was a bugfix actually because it would never return `this.volume = 0` (resolves to `false`, unless `this.media.volume` was coincidentally 0 as well)